src/resources/css/rsvp.css

```diff
@@ -187,4 +187,18 @@
 .event-tickets-meta-label {
 	font-weight:700;
 	margin: 0 1em 0 0;
-}
\ No newline at end of file
+}
+
+/* TwentySixteen fix for box shadow as bottom border on <a> link */
+
+.entry-content a,
+.entry-summary a,
+.taxonomy-description a,
+.logged-in-as a,
+.comment-content a,
+.pingback .comment-body > a,
+.textwidget a,
+.entry-footer a:hover,
+.site-info a:hover {
+	box-shadow: none;
+}
```

Comment:
If this block is only targeting Twenty Sixteen should we make use of `.tribe-theme-twentysixteen` body class in the selectors? (If we are happy to use them with all themes that's fine).